### ISSUE TITLE
Detekt 1.16.0 and newer cannot be downloaded due to missing artifact

### ISSUE THREAD

**Issue by psud** [natiginfo/action-detekt-all#23]
Hello all,

We use the detekt GitHub action as follows (shortend to relevant parts only):
```
jobs:
  detekt:
    name: run detekt analysis
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Code
        uses: actions/checkout@v2
      - name: Detekt
        uses: natiginfo/action-detekt-all@1.16.0
        with:
          args: '-c ${{ env.detekt_config }} -r html:${{ env.detekt_report }}
```

When GitHub executes the action it fails as it cannot download the artifact from the expected directory
```
Step 2/5 : ADD https://github.com/detekt/detekt/releases/download/v1.16.0/detekt /usr/local/bin/detekt
  ADD failed: failed to GET https://github.com/detekt/detekt/releases/download/v1.16.0/detekt with status 404 Not Found:
<!DOCTYPE html>
  <html> ... </html>
```

Accessing the release download manually also does not work for the newer releases but it does work for old releases
https://github.com/detekt/detekt/releases/download/v1.16.0/detekt --> NOT_FOUND
https://github.com/detekt/detekt/releases/download/v1.16.0-RC1/detekt --> NOT_FOUND
https://github.com/detekt/detekt/releases/download/v1.16.0-RC-2/detekt --> NOT_FOUND
https://github.com/detekt/detekt/releases/download/v1.16.0-RC-3/detekt --> NOT_FOUND
https://github.com/detekt/detekt/releases/download/v1.17.0-RC1/detekt --> NOT_FOUND
https://github.com/detekt/detekt/releases/download/v1.15.0/detekt --> OK

Looking at the assets in for v 1.15.0 we can se a "detekt" asset (see https://github.com/detekt/detekt/releases/tag/v1.15.0) but for v1.16.0 this asset does not exist (see https://github.com/detekt/detekt/releases/tag/v1.16.0) .

This issue does seem similar to https://github.com/natiginfo/action-detekt-all/issues/19 but on closer inspection I think the problem source is different.

Thanks in advance for your input!

**Comment by natiginfo** [comment]
Hi,

Thank you for reporting the issue.

Indeed, it seems like it is same issue as #19. I fixed releases for `1.16.0` and `1.17.0-RC-1` and tested in my project. I will make releases for `1.16.0-RC1`, `1.16.0-RC-2` and `1.16.0-RC-3` later (just in case someone uses them) but for now other releases should be ok.

Please, test it in your end also and let me know if there is new issue.

I will monitor upcoming releases and try to prevent this type of problems in case `release-manager` automation does not work.

**Comment by psud** [comment]
Thanks for the quick response! Everything now works on my end. Your work is very much appreciated!
